client/src/login/loginErrorFromStatusCode.ts

```
@@ -0,0 +1,11 @@
+export const loginErrorFromStatusCode = (code: number): string => {
```

I would return an error message from the server together with a status code so the client can display it as is.